<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>im</class>
 <widget class="QMainWindow" name="im">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageProcessingKit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_in">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Input Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_out">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Output Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView_in">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="graphicsView_out">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_coord">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_color_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Close"/>
    <addaction name="action_Print"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Reset"/>
    <addaction name="action_Histogram"/>
    <addaction name="action_Grayscale"/>
   </widget>
   <widget class="QMenu" name="menuColor_Space">
    <property name="title">
     <string>&amp;Color Space</string>
    </property>
    <addaction name="action_Adjust_HSV"/>
   </widget>
   <widget class="QMenu" name="menuSpatial_Transform">
    <property name="title">
     <string>&amp;Gray Transform</string>
    </property>
    <addaction name="action_Linear_Transformation"/>
    <addaction name="action_Piecewise_Linear_Transformation"/>
    <addaction name="action_Histogram_Equalization"/>
    <addaction name="action_Histogram_Specification"/>
   </widget>
   <widget class="QMenu" name="menuSpatial_Filter">
    <property name="title">
     <string>&amp;Spatial Filter</string>
    </property>
    <addaction name="action_Median_Filter"/>
    <addaction name="action_Maximum_Filter"/>
    <addaction name="action_Minimum_Filter"/>
    <addaction name="action_Custom_Filter"/>
    <addaction name="action_Average_Filter"/>
    <addaction name="action_Laplacian_Filter"/>
    <addaction name="action_Pseudocolor"/>
   </widget>
   <widget class="QMenu" name="menuFrequency_Filter">
    <property name="title">
     <string>Freq&amp;uency Filter</string>
    </property>
    <addaction name="action_Low_Pass_Filter"/>
    <addaction name="action_Ideal_High_Pass_Filter"/>
    <addaction name="action_Butterworth_Low_Pass_Filter"/>
    <addaction name="action_Butterworth_High_Pass_Filter"/>
   </widget>
   <widget class="QMenu" name="menuImage_Recovery">
    <property name="title">
     <string>Image &amp;Recovery</string>
    </property>
    <addaction name="action_Inverse_filter"/>
   </widget>
   <widget class="QMenu" name="menuImage_Thresholding">
    <property name="title">
     <string>Image &amp;Thresholding</string>
    </property>
    <addaction name="action_Manual_Threshold"/>
    <addaction name="action_Ostu_method"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_ImageProcessingKit"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuColor_Space"/>
   <addaction name="menuSpatial_Transform"/>
   <addaction name="menuSpatial_Filter"/>
   <addaction name="menuFrequency_Filter"/>
   <addaction name="menuImage_Recovery"/>
   <addaction name="menuImage_Thresholding"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Restore</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Histogram">
   <property name="text">
    <string>&amp;Histogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Grayscale">
   <property name="text">
    <string>&amp;Grayscale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Adjust_HSV">
   <property name="text">
    <string>Adjust HSV</string>
   </property>
  </action>
  <action name="action_Linear_Transformation">
   <property name="text">
    <string>&amp;Linear Transformation</string>
   </property>
   <property name="toolTip">
    <string>Linear Transformation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Piecewise_Linear_Transformation">
   <property name="text">
    <string>Piecewise Linear Transformation</string>
   </property>
   <property name="toolTip">
    <string>Piecewise Linear Transformation</string>
   </property>
  </action>
  <action name="action_Histogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
   <property name="toolTip">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="action_Median_Filter">
   <property name="text">
    <string>Median Filter</string>
   </property>
   <property name="toolTip">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="action_Maximum_Filter">
   <property name="text">
    <string>Maximum Filter</string>
   </property>
   <property name="toolTip">
    <string>Maximum Filter</string>
   </property>
  </action>
  <action name="action_Minimum_Filter">
   <property name="text">
    <string>Minimum Filter</string>
   </property>
   <property name="toolTip">
    <string>Minimum Filter</string>
   </property>
  </action>
  <action name="action_Custom_Filter">
   <property name="text">
    <string>Custom Filter</string>
   </property>
   <property name="toolTip">
    <string>filter with custom kernel</string>
   </property>
  </action>
  <action name="action_Low_Pass_Filter">
   <property name="text">
    <string>Ideal Low Pass Filter</string>
   </property>
   <property name="toolTip">
    <string>Ideal Low Pass Filter</string>
   </property>
  </action>
  <action name="action_Ideal_High_Pass_Filter">
   <property name="text">
    <string>Ideal High Pass Filter</string>
   </property>
   <property name="toolTip">
    <string>Ideal High Pass Filter</string>
   </property>
  </action>
  <action name="action_Butterworth_Low_Pass_Filter">
   <property name="text">
    <string>Butterworth Low Pass Filter</string>
   </property>
   <property name="toolTip">
    <string>Butterworth Low Pass Filter</string>
   </property>
  </action>
  <action name="action_Butterworth_High_Pass_Filter">
   <property name="text">
    <string>Butterworth High Pass Filter</string>
   </property>
   <property name="toolTip">
    <string>Butterworth High Pass Filter</string>
   </property>
  </action>
  <action name="action_Inverse_filter">
   <property name="text">
    <string>Inverse Filter</string>
   </property>
   <property name="toolTip">
    <string>Inverse Filter</string>
   </property>
  </action>
  <action name="action_Manual_Threshold">
   <property name="text">
    <string>Manual Threshold</string>
   </property>
   <property name="toolTip">
    <string>Manual Threshold</string>
   </property>
  </action>
  <action name="action_Ostu_method">
   <property name="text">
    <string>Ostu's method</string>
   </property>
  </action>
  <action name="action_About_ImageProcessingKit">
   <property name="text">
    <string>About ImageProcessingKit</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_Histogram_Specification">
   <property name="text">
    <string>Histogram Specification</string>
   </property>
   <property name="toolTip">
    <string>Histogram Specification</string>
   </property>
  </action>
  <action name="action_Average_Filter">
   <property name="text">
    <string>Average Filter</string>
   </property>
   <property name="toolTip">
    <string>Average Filter</string>
   </property>
  </action>
  <action name="action_Laplacian_Filter">
   <property name="text">
    <string>Laplacian Filter</string>
   </property>
   <property name="toolTip">
    <string>Laplacian Filter</string>
   </property>
  </action>
  <action name="action_Pseudocolor">
   <property name="text">
    <string>Pseudocolor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
